LICENSE.txt

```diff
@@ -0,0 +1 @@
+see COPYING
```

Comment:
I would agree on that, but do not know the reason for it to be in COPYING...

I would leave for @sde1000 to comment on this :)